private/topoplot_common.m

```diff
@@ -52,13 +54,17 @@
 % the call with multiple inputs is done by ft_topoplotIC and recursively by ft_topoplotER/TFR itself
 % the last input argument points to the specific data to be plotted
 if Ndata>1 && ~isnumeric(varargin{end})
+  
   for k=1:Ndata
     
-    if k>1
+    if k>1 && istrue(cfg.newfigure)
```

Comment:
please integrate this as
if k>1
  if istrue(cfg.newfigure)
    ...
  elseif ~istrue(cfg.newfigure)  
    ...
  end
end 

How will this play with interactive figures?

Reply:
seems just fine. when selecting some channels in one of the subplots it proceeds to a singleplotER representation.